src/sentry/static/sentry/app/utils.tsx

```diff
@@ -225,21 +226,29 @@ export const buildTeamId = id => `team:${id}`;
 /**
  * Removes the organization / project scope prefix on feature names.
  */
-export function descopeFeatureName(feature) {
-  return typeof feature.match !== 'function'
-    ? feature
-    : feature.match(/(?:^(?:projects|organizations):)?(.*)/).pop();
+export function descopeFeatureName(feature: string): string {
```

Comment:
This seems like it's not always a string

Reply:
where is it not a string?

Reply:
If it's a string then line 222 - 224 doesn't make sense

Reply:
Ah hm, I forget why I wrote it like that 😬

Reply:
I've adjusted the types for this function